crontab

```diff
@@ -0,0 +1,2 @@
+# renew certificate around every midnight and noon, if necessary
+0 0,12 * * * sleep 3109 && /home/monitoring/renew-cert.sh > /home/monitoring/last_cert_renewal.log 2>&1

```

Comment:
There is no separation between production and test here. When I understand the README correctly, all this acme stuff is done for both separately. Correct? Or is the intention to move all this to the reverse proxy and do it once, only?

Reply:
Right now we only have one app running under both domains. Therefore we issue and renew a single certificate (see README: " We currently use a single TLS certificate for both ...").
I'm not sure yet how exactly all of this will work when we add production and the extra nginx, but we're quite flexible there.